### PR TITLE
Added Contributors to readme.MD

### PR DESCRIPTION
**Pull request by martinisiu**
Had to add some changes so thought I'd just make a contributors list

### CODE REVIEW

File: readme.MD [hunk 1]
```diff
@@ -18,3 +18,8 @@ Default bases include (landing-page, portfolio-page)
 Components: components are children of Bases. A Component must be a child of at least one Base.
 Components are invoked via --flags.
 Default components include (navbar, navbar-searchbar, sidebar, footer, footer-big, --background-video)
+
+
+Contributors:
```
Comment (ZenaMel) [inline readme.MD]: This should be a header no? Like so:
# Contributors:

**Review by ZenaMel — CHANGES_REQUESTED** [review]
Contributors: should be a header and not just a paragraph.
For reference on markdown notation: https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax

File: readme.MD (hunk 1 above)
Comment (martinisiu) [reply readme.MD]: This is true

**Comment by martinisiu** [comment]
Fixed it and added links to each github

**Review by ZenaMel — APPROVED** [review]
nice work! Looking good.

**Pull request merged**